src/fe-common/core/printtext.c

```diff
@@ -404,7 +405,18 @@ static void msg_beep_check(TEXT_DEST_REC *dest)
 	    (beep_when_away || (dest->server != NULL &&
 				!dest->server->usermode_away)) &&
 	    (beep_when_window_active || dest->window != active_win)) {
-                signal_emit("beep", 0);
+		if (beep_msg_level_ignore & MSGLEVEL_HIDDEN) {
```

Comment:
We'd be able to skip this PR just by adding the check for MSGLEVEL_HIDDEN to line https://github.com/irssi/irssi/blob/80926476fc4c17dea9f24f9da753f7443b8c05e1/src/fe-common/core/printtext.c#L403
ie
`(dest->level & (MSGLEVEL_NO_ACT|MSGLEVEL_HIDDEN)) == 0`
if I remember my c correctly, is there any other use case for this PR?

Reply:
we maybe don't want to abort the beep if the user has _not_ hidden the hidden lines (using /window hidelevel -hidden); and we may want to abort the beep if the user has hidden another line (e.g. using /window hidelevel +joins)

Reply:
Hum hum, yeah